src/context/wishContext.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext, createContext, useReducer } from "react";
+import { wishReducer } from "../reducers/wishReducer";
+
+const wishListContext = createContext();
+
+const WishListProvider = ({ children }) => {
+  const [wishState, wishDispatch] = useReducer(wishReducer, {
+    wishCount: 0,
```

Comment:
alternative way - in return section just add - <span> {wishlist.length} </span> . so , don't need to add wishCount:0.

Reply:
wow, good suggestion.